entities.c

```diff
@@ -390,3 +390,32 @@ size_t decode_html_entities_utf8(char *dest, const char *src)
 	return (size_t)(to - dest);
 }
 
+int encode_html_entities(char *dest, const char *src) {
+        char *to = dest;
+        for( const char *from = src ; *from ; from++ ) {
+            int i = 9999;
```

Comment:
Please declare `i` as late as possible and do not assign a magic unused value to it

Reply:
Completely agree. I'll commit the correction.